dev/tools/localization/bin/gen_date_localizations.dart

```diff
@@ -123,28 +133,67 @@ Future<void> main(List<String> rawArgs) async {
   if (writeToFile) {
     final File dateLocalizationsFile = File(path.join('packages', 'flutter_localizations', 'lib', 'src', 'l10n', 'generated_date_localizations.dart'));
     dateLocalizationsFile.writeAsStringSync(buffer.toString());
-    Process.runSync(path.join('bin', 'cache', 'dart-sdk', 'bin', 'dartfmt'), <String>[
```

Comment:
This no longer exists

Reply:
We may be missing a test here that should have failed when this was removed?